kyo-combinators/shared/src/main/scala/kyo/AsyncCombinators.scala

```diff
@@ -7,24 +7,34 @@ import scala.annotation.tailrec
 import scala.annotation.targetName
 import scala.util.NotGiven
 
-extension [A, E, Ctx](effect: A < (Abort[E] & Async & Ctx))
+extension [A, E, S](effect: A < (Abort[E] & Async & S))
```

Comment:
We can probably get rid of the `Abort[E]` term here, since Fiber is no longer bound up with E. I can follow up separately if you don't want to go through that in this PR.

Reply:
thanks!